CertificationAuthority/src/CryptoCAMain.java

```diff
@@ -0,0 +1,55 @@
+import static io.github.olivierlemasle.ca.CA.*;
+import java.security.PublicKey;
+import io.github.olivierlemasle.ca.*;
+
+public class CryptoCAMain {
+    public static void main(String[] args) {
+        // vytvoreni ca root certifikatu
+        final DistinguishedName root = dn("CN=Root-CustomCA, O=CustomCA");
+        final RootCertificate ca = createSelfSignedCertificate(root)
+            .validDuringYears(10)
+            .build();
+        
+        ca.saveKey("CACert.key");
+        ca.save("CACert.crt");
+
+        // zadost o klient. certifikat
+        final CsrWithPrivateKey csr = createCsr().generateRequest(
+                dn()
+                .setCn("*.navalproject.online")
+                .setO("EncryptedStorage")
+                .setOu("undefined dept.")
+                .setSt("Moravsky")
+                .setC("CZ")
+                .setAlternativeName("DNS Name=*.navalproject.online")
+                .build());
+        
+        // podepsani klient. certifikatu
+        final Certificate cert = ca.signCsr(csr)
+                .setRandomSerialNumber()
+                .validDuringYears(2)
+                .sign();
+        
+        System.out.println(csr.getPrivateKey());
+        cert.save("ServerCert.crt");
+        final CertificateWithPrivateKey priv = cert.attachPrivateKey(csr.getPrivateKey());
```

Comment:
You can use full length variables naming (priv -> certificateWithPrivateKey). That way is more readable, thanks.